src/main/java/ee/aktors/misp2/configuration/DigiDoc4jConfiguration.java

```diff
@@ -54,6 +54,7 @@
     private static final String PARAM_TEST_MODE = "digidoc4j.test";
     private static final String PARAM_OCSP_SOURCE = "digidoc4j.ocsp";
     private static final String PARAM_TRUSTED_TERRITORIES = "digidoc4j.trustedTerritories";
+    private static final String PARAM_MID_ENABLED = "auth.mobileID";
```

Comment:
Currently, the `auth.mobileID` configuration parameter is not mentioned in the installation guide. It should be added to the `5.4.1 Service parameters` section.